### PR TITLE
Feature/main page

### PR DESCRIPTION
**Pull request by kvnloughead**
Ok, the changes you suggested have been made.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,16 +1,205 @@
...
-    <li><img src="images/search.svg" alt="search icon"></li>
-  </ul>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/403e626d8b.js"
```
Comment (momofcats) [inline index.html:7]: we don't need fontawesome anymore

File: index.html [hunk 2]
```diff
@@ -1,16 +1,205 @@
...
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="blocks/index.css" />
+    <title>Supermarket App</title>
+  </head>
+  <body class="page">
+    <ul class="nav">
```
Comment (momofcats) [inline index.html:14]: I was thinking to move this nav in header and give it a class name "main-nav". This also needs to be wrapped in nav tag.

File: index.html [hunk 3]
```diff
@@ -1,16 +1,205 @@
...
+      <li><img src="images/menu.svg" alt="menu icon" /></li>
+      <li><img src="images/search.svg" alt="search icon" /></li>
+    </ul>
+    <header class="header">
+      <ul class="header__category-list">
```
Comment (momofcats) [inline index.html:19]: This also needs nav tag. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav

File: index.html [hunk 4]
```diff
@@ -1,16 +1,205 @@
...
+        <li><a class="header__category" href="#">Bakery</a></li>
+        <li><a class="header__category" href="#">Sandwiches</a></li>
+        <li><a class="header__category" href="#">Dairy</a></li>
+        <li><a class="header__category" href="#">Meat</a></li>
+        <li><a class="header__category" href="#">Fish</a></li>
+      </ul>
+    </header>
+    <main>
+      <ul class="columns">
+        <div class="column">
```
Comment (momofcats) [inline index.html:35]: This list doesn't make any sense if you have div class="column". Did you forget to delete it? Your previous implementation was layout with flex.

File: index.html (hunk 1 above)
Comment (kvnloughead) [reply index.html:7]: Fixed.

File: index.html (hunk 2 above)
Comment (kvnloughead) [reply index.html:14]: Good call here, and we'll take care of this after merging.

File: index.html (hunk 3 above)
Comment (kvnloughead) [reply index.html:19]: You're right, I'm always forgetting that.   Let's leave this until we fix the rest of the header.

File: index.html (hunk 4 above)
Comment (kvnloughead) [reply index.html:35]: Oh yeah, I guess I did forget.   It's fixed.

**Pull request merged**